_pytest/capture.py

```diff
@@ -172,7 +172,7 @@ def capfd(request):
     captured output available via ``capfd.readouterr()`` method calls
     which return a ``(out, err)`` tuple.
     """
-    if "capsys" in request._funcargs:
+    if "capsys" in request.fixturenames:
```

Comment:
note to self: add a test here that this will work when capsys is called using `request.getfixturevalue()`